scripts/upload.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Example script that uploads all paths passed in as CLI arguments.
+ */
+
+const process = require("process");
+
+const { SkynetClient } = require("..");
+
+const client = new SkynetClient();
+
+const promises = process.argv.slice(2).map((path) => client.uploadFile(path));
+(async () => {
+  const skylinks = await Promise.all(promises);
```

Comment:
`Promise.all` rejects immediately. I think it would be better to use `Promise.allSettled`.